package.json

```
@@ -3,7 +3,7 @@
 	"publisher": "biomejs",
 	"displayName": "Biome",
 	"description": "Biome LSP VS Code Extension",
-	"version": "2.2.2",
+	"version": "2.2.3",
```

```suggestion
	"version": "2.2.2",
```

I think we leave this untouched, just in case we need to merge another fix in the meantime. I'll update prior to release anyway.

---

Yeah sorry, I didn't want to bump it